packages/app-mobile/services/e2ee/crypto.ts

```diff
@@ -0,0 +1,49 @@
+import { Crypto, CryptoBuffer } from '@joplin/lib/services/e2ee/types';
+import QuickCrypto from 'react-native-quick-crypto';
+import { HashAlgorithm } from 'react-native-quick-crypto/lib/typescript/keys';
+
+const crypto: Crypto = {
+
+	getCiphers: (): string[] => {
+		return QuickCrypto.getCiphers();
+	},
+
+	getHashes: (): string[] => {
+		return QuickCrypto.getHashes();
+	},
+
+	randomBytes: async (size: number): Promise<CryptoBuffer> => {
+		return new Promise((resolve, reject) => {
+			QuickCrypto.randomBytes(size, (error, result) => {
+				if (error) {
+					reject(error);
+				} else {
+					resolve(result);
+				}
+			});
+		});
+	},
+
+	pbkdf2Raw: async (password: string, salt: CryptoBuffer, iterations: number, keylen: number, digest: string): Promise<CryptoBuffer> => {
+		const digestMap: { [key: string]: HashAlgorithm } = {
```

Comment:
Again please read the coding style guide, we explicitly ask not to set types this way. See https://joplinapp.org/help/dev/coding_style#avoid-inline-types

I really shouldn't have to keep reminding you to read it. It's understandable to miss a few details here and there but for this PR that's what, the third or fourth time that I remind you to read it?

Reply:
Sorry I only checked the method declaration. I will change it later.